src/index.js

```diff
@@ -4,5 +4,6 @@ import './index.css';
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 
+window.assemble = {}
```

Comment:
what does this line do?

Reply:
Our solution briefly stores the scroll position
as a global variable as the event fires,
so it can be reset by another callback once the event ends.

Specifically,
we're storing the scroll position as `window.assemble.scroll`.
This line merely primes the application
so we're sure that `window.assemble` exists.

The callbacks happen one after another,
immediately off of the JS execution stack,
so there is no visual indication of scrolling.